website/docs/08_glossary/01_devops.mdx

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
All pages glossary entries should not be in the docs/ folder but rather in the glossary folder, so the url of a glossary entry is: not `http://localhost:3000/docs/glossary/devops/` but `http://localhost:3000/glossary/devops/`

Reply:
This is done